### PR TITLE
Adds Cult Stun Nerf to Antagonist Tips

### PR DESCRIPTION
**Pull request by SpaceDragon00**
## About The Pull Request

Adds a new line to the cultist antag tips that details the stun nerf present on fulpstation

image

## Why It's Good For The Game

Better clarity for custom nerfs; provides information that isn't present on the referenced wiki

## Changelog
:cl:
qol: Cult antag tips now mention the stun nerf
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
I don't like how this is added. This is explained in the cult stun arm's description, anyways.

**Comment by SpaceDragon00** [comment]
> 
> 
> I don't like how this is added. This is explained in the cult stun arm's description, anyways.

Nothing wrong with redundancy, especially when they may be operating on the assumption that it works on everyone, due to the nerf not being on the main cult reference page

**Review by SgtHunk — APPROVED** [review]
This is fine by me. The cult stun nerf is on our end, and it's balance is significant enough (due to stuns being one of the cult's main power) that it should be mentioned. The tips mentioning it is better than a single tooltip/description that references it.

**Comment by remove32** [comment]
i still am not a huge fan of how these pages show and the code that does it but i'll allow it for now because i dont have the willpower or time to argue about it

**Comment by SgtHunk** [comment]
> i still am not a huge fan of how these pages show and the code that does it but i'll allow it for now because i dont have the willpower or time to argue about it

don't worry, I'll convert this to TGUI when biddle traitors gets merged

**Comment by SgtHunk** [comment]
Not yet, I could make a verb for it though.
And/or wait for biddle since it adds an action button for it iirc

File: fulp_modules/features/antag_tips/html/cultist.html [hunk 1]
```diff
@@ -9,6 +9,7 @@ <h1>You are a Blood Cultist!</h1>
 	<p><img src="sacrune.png"/>To begin, you should gather converts by placing them over an <b>Offering Rune.</b></p>
 	<p>There will be several other cultists on board, communicate with them using the commune button. Find a place to create a base, and keep cult structures hidden.</p>
 	<p>Before you can summon Nar'Sie, you must first sacrifice a target for her on an offering rune. Check the top right of your screen for information on your target.</p>
+	<p><font size="+1"><b>Note that your stun spell will not stun those with shielded minds-- consider alternative tactics for dealing with them.</b></font></p>
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antag_tips/html/cultist.html:12]:
```suggestion
	<p><b>NOTE: Your Cult Stun spell will not have full effect on Mindshielded people -- consider using alternative tactics when dealing with them.</b></p>
```

**Comment by remove32** [comment]
> 
> 
> Not yet, I could make a verb for it though.
> And/or wait for biddle since it adds an action button for it iirc

whats biddle

**Comment by SgtHunk** [comment]
biddle traitors, a PR that makes traitors progression-based
It comes alongside TGUI panel integrations that show flavor text, objectives, etc

**Pull request merged**